service/src/test/java/bio/terra/cbas/dao/TestMethodDao.java

```diff
@@ -70,16 +70,13 @@ passing in a current_timestamp() value, causing the test to fail.
   @Test
   void retrievesAllMethods() {
     List<Method> allMethods = methodDao.getMethods();
-    assertEquals(5, allMethods.size()); // because database has 3 pre-staged workflows
+    assertEquals(2, allMethods.size());
```

Comment:
Great question... what I ended up doing was creating a separate cbas database instance for unit testing vs actually interacting with it (back when we could test changes against local cbas-ui). I think the problem is that these tests assume a clean setup so if you've imported any methods or ran any workflows using the local instance it fails on those tests. So maybe if you re-initialize or create a new version of your databases it will work? But also, maybe not, because some of the tests here don't clean up after themselves (looking at RunSetDao).

Reply:
If RunSetDao is given the ability to delete run sets we should be able to clean up properly 🤔 perhaps worth a KTLO type ticket.